go-controller/pkg/clustermanager/userdefinednetwork/template/net-attach-def-template.go

```diff
@@ -203,6 +257,22 @@ func validateIPAM(ipam *userdefinednetworkv1.IPAMConfig) error {
 	return nil
 }
 
+func validateVLANConfig(vlanConfig *userdefinednetworkv1.VLANConfig) error {
+	if vlanConfig == nil {
+		return nil
+	}
+	if vlanConfig.Mode != userdefinednetworkv1.VLANModeAccess {
+		return fmt.Errorf("vlan.mode support 'Access' mode only")
```

Comment:
nit, maybe better error message is: invalid vlan.mode value. Only current acceptable value is 'Access'